test/cases/ostree.sh

```diff
@@ -108,7 +108,7 @@ case "${ID}-${VERSION_ID}" in
             BOOT_LOCATION="${COMPOSE_URL:-}/compose/BaseOS/x86_64/os/"
         fi
         ;;
-    "rhel-9.3")
+    "rhel-9.4")
         IMAGE_TYPE=edge-commit
```

Comment:
Another comment: not sure why we've kept references to previous minor releases but I think you need to do either one of these:

1) Remove them to keep this file clean or 
2) Keep the existing entries and add new ones for the new distros

Reply:
Yeah it's probably redundant as we're only running this test on the latest nightly in the edge-ci so I'll remove all the older entries.